### ISSUE TITLE
Doesn't work with guard 2.13

### ISSUE THREAD

**Issue by dsandstrom** [jacquescrocker/guard-resque#17]
There were some breaking changes to [guard plugin](https://github.com/guard/guard/wiki/Create-a-guard) loading.  To start with you have to use:
```
require "guard/compat/plugin"

module Guard
  class Resque < Plugin
```
instead of:
```
require 'guard'
require 'guard/plugin'
require 'timeout'

module Guard
  class Resque < Plugin
```

**Comment by pbibalan** [comment]
+1. @jacquescrocker any updates on this?

**Comment by jacquescrocker** [comment]
@pbibalan @dsandstrom sorry! i wasnt paying attention.

@pbibalan can you test the PR? if it works for you, i'll merge

happy to provide commit access to anyone who wants it. i dont actually use resque anymore

**Comment by nestorw** [comment]
This PR works for me using guard 2.14. Can we get it merged?

**Comment by jacquescrocker** [comment]
@nestorw @dsandstrom i added you as collaborators. feel free to merge

let me know your emails too and i can add you to the rubygem also

**Comment by dsandstrom** [comment]
Closed by #18
